drivers/hmis/app/models/hmis/hud/validators/custom_assessment_validator.rb

```diff
@@ -0,0 +1,17 @@
+class Hmis::Hud::Validators::CustomAssessmentValidator < Hmis::Hud::Validators::BaseValidator
+  IGNORED = [
+    :ExportID,
+    :DateCreated,
+    :DateUpdated,
+    # CE fields are not present on the custom assessment
+    :AssessmentID,
+    :AssessmentLocation,
+    :AssessmentType,
+    :AssessmentLevel,
+    :PrioritizationStatus,
+  ].freeze
+
+  def configuration
+    Hmis::Hud::CustomAssessment.hmis_configuration(version: '2022').except(*IGNORED)
```

Comment:
Where is `hmis_configuration` defined for the context of `CustomAssessmen`?

Reply:
I set it up to use the Assessment HMIS configuration, and then have the validator ignore all the CE fields

Reply:
It's defined with `include ::HmisStructure::Assessment` on the CustomAssessment

Reply:
got it.  Thanks